packages/headless/src/modules/user/services/user-profile.service.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpStatus, Injectable } from '@nestjs/common';
+import { HelperService } from 'src/helper/helper.service';
+import { UserRepository } from '../repository';
+
+@Injectable()
+export class UserProfileService {
+  constructor(
+    private userRepo: UserRepository,
+    private helperService: HelperService,
+  ) {}
+
+  async getProfile(userId: string) {
+    const user = await this.userRepo.findOne({ _id: userId });
```

Comment:
we don't use mongoose _id. Please change this .

Reply:
removed